cpp/tests/io/external_datasource_test.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+
+#include <cudf/cudf.h>
+#include <nvstrings/NVStrings.h>
+#include <cudf/legacy/unary.hpp>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <tests/utilities/legacy/cudf_test_fixtures.h>
```

Comment:
Don't use the legacy fixture, it is soon to be removed.
```suggestion
#include <tests/utilities/base_fixture.hpp>
```